lib/rubocop/cop/mixin/ignored_methods.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    # This module encapsulates the ability to ignore certain methods when
+    # parsing.
+    module IgnoredMethods
```

Comment:
Good call on making this a reusable module.

Reply:
Thanks. I was surprised at the number of ways to check the same config in different cops (with `.to_a` or without, `map(&:to_sym)` or `name.to_s`).